docs/estimators/index.rst

```diff
@@ -19,10 +19,10 @@ energy.
 The core of any estimator algorithm is hypothesis testing: a reference
 model or counts excess is tested against a null hypothesis. From the
 best fit reference model a flux is derived and a corresponding :math:`\sqrt{\Delta TS}`
-value from the difference in fit statistics to the null hypothesis,
-assuming one degree of freedom. In this case
-:math:`\sqrt{\Delta TS}` represents an approximation of the
-"classical significance".
+value from the difference in fit statistics to the null hypothesis.
+Assuming one degree of freedom, :math:`\sqrt{\Delta TS}` represents an
+approximation (`Wilk's theorem <https://en.wikipedia.org/wiki/Wilks%27_theorem>`_)
+of the "classical significance".
```

Comment:
Is it an approximation of the significance or its definition in our domain?

Reply:
Hm, I guess it's both and there is no either / or? Taken strictly it's an approximation according to Wilk's theorem and it's also the widely used definition in our domain...